bitnami/ghost/templates/deployment.yaml

```diff
@@ -154,10 +154,12 @@ spec:
               value: {{ .Values.ghostBlogTitle | quote }}
             - name: GHOST_ENABLE_HTTPS
               value: {{ ternary "yes" "no" .Values.ghostEnableHttps | quote }}
-            - name: GHOST_EXTERNAL_HTTP_PORT_NUMBER
-              value: {{ coalesce .Values.service.ports.http .Values.service.port| quote }}
-            - name: GHOST_EXTERNAL_HTTPS_PORT_NUMBER
-              value: {{ coalesce .Values.service.ports.https .Values.service.httpsPort| quote }}
+            {{- if eq ( toString ( .Values.service.type )) "LoadBalancer" }}
+              - name: GHOST_EXTERNAL_HTTP_PORT_NUMBER
+                value: {{ coalesce .Values.service.ports.http .Values.service.port| quote }}
+              - name: GHOST_EXTERNAL_HTTPS_PORT_NUMBER
+                value: {{ coalesce .Values.service.ports.https .Values.service.httpsPort| quote }}
```

Comment:
Wrong indentation
```suggestion
            - name: GHOST_EXTERNAL_HTTP_PORT_NUMBER
              value: {{ coalesce .Values.service.ports.http .Values.service.port| quote }}
            - name: GHOST_EXTERNAL_HTTPS_PORT_NUMBER
              value: {{ coalesce .Values.service.ports.https .Values.service.httpsPort| quote }}
```

Reply:
thank you for your concerns!
as far as i can recall, i have not resolved the https passthrough issue. I have indeed had to resort to tls termination, because of this.

this PR adresses the kubernetes  preassumption that https means 443, which would ignore any customization on your end.

Reply:
I see. I think we should try to address the root problem here, the tls termination. We can apply your changes but I don't see any value if the problem persist

Reply:
tls termination is not that uncommon for reverse proxies
but it is up to a different issue though, this only solves configuration recognition issues if you try to set a different port from the default.

Reply:
@fmulero you can open a new issue were we are able to address the details.
as i am still interested in https passthrough, but i got stuck on the container configuration.

it told the the address it tried to bind to was in use.